Tests/Amr/Advection_AmrCore/Exec/inputs

```diff
@@ -2,7 +2,7 @@
 # Run until nsteps == max_step or time == stop_time, 
 #     whichever comes first
 # *****************************************************************
-max_step  = 1000
+max_step  = 100
```

Comment:
Can we keep the current inputs file and add a new file for the bittree test?

Reply:
Done